iocore/eventsystem/I_VConnection.h

```diff
@@ -116,13 +116,16 @@
 //
 //////////////////////////////////////////////////////////////////////////////
 
-/** Used in VConnection::shutdown(). */
-enum ShutdownHowTo_t {
-  IO_SHUTDOWN_READ = 0,
-  IO_SHUTDOWN_WRITE,
-  IO_SHUTDOWN_READWRITE,
+enum class IOShutdown {
+  READ      = SHUT_RD,
+  WRITE     = SHUT_WR,
+  READWRITE = SHUT_RDWR,
 };
 
+constexpr IOShutdown IO_SHUTDOWN_READ      = IOShutdown::READ;
+constexpr IOShutdown IO_SHUTDOWN_WRITE     = IOShutdown::WRITE;
+constexpr IOShutdown IO_SHUTDOWN_READWRITE = IOShutdown::READWRITE;
```

Comment:
At least add a comment to the effect this is for compatibility and should be considered transitory until we can do a full cleanup.